src/Solver/Numerov.cpp

```diff
@@ -0,0 +1,249 @@
+#include "Numerov.h"
+
+static inline double square (double x) { return x*x; }
+
+Numerov::Numerov(Potential potential, int nbox)
+{
+    this->potential        = &potential;
+    this->nbox             = nbox;
+    this->solutionEnergy   = 0;
+    this->probability      = std::vector<double>(nbox+1);
+    this->wavefunction     = std::vector<double>(nbox+1);
+    this->tempWavefunction = std::vector<double>(nbox+1);
+
+    // Fill wavefunction with 1 in order to multiply each wavefunction found on separated potentials
+    std::fill(this->wavefunction.begin(), this->wavefunction.end(), 1.0);
+    std::fill(this->tempWavefunction.begin(), this->tempWavefunction.end(), 0.0);
+    this->tempWavefunction[0] = 0.0;
+    this->tempWavefunction[1] = 0.1;
```

Comment:
This initialisation does not belong to the start of class, but to something that should be called before `findWavefunction`. That is, the setup of the boundary conditions, that should be in `solve` and `bisection`.
Now bisection is too coupled with solved, so it cannot be generalized, so there is fine. (cf. later comment)

Reply:
We can implement a method to specify the boundary conditions, so it's even more general. (need issue maybe)